### PR TITLE
CIV-2911

### PR DESCRIPTION
**Pull request by MMNycz**
Add new templates for  application type STRIKE_OUR -> DAMAGES, OCMC.

**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-2911

### Change description ###

Add new notification templates for  application type STRIK_OUR -> DAMAGES, OCMC.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java [hunk 1]
```diff
@@ -224,20 +225,48 @@ private void applicationRequestForInformation(CaseData caseData) {
     }
 
     private void applicationApprovedNotification(CaseData caseData) {
-        if (areRespondentSolicitorsPresent(caseData)) {
+        if (areRespondentSolicitorsPresent(caseData)
+            && useDamageTemplate(caseData)) {
+            caseData.getGeneralAppRespondentSolicitors().forEach(
```
Comment (karthick-mohanasundaram-HMCTS) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java:230]: Could we have a method call as the same block of code has been used in multiple places if that's applicable?

caseData.getGeneralAppRespondentSolicitors().forEach(
                respondentSolicitor -> sendNotificationForJudicialDecision(
                    caseData, respondentSolicitor.getValue().getEmail(),
                    notificationProperties.getJudgeApproveOrderToStrikeOutDamages()
                ));

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java [hunk 2]
```diff
@@ -312,9 +341,19 @@ private void applicationDirectionOrder(CaseData caseData) {
     }
 
     private void judgeApprovedOrderApplicationCloak(CaseData caseData) {
-        sendNotificationForJudicialDecision(caseData,
-            caseData.getGeneralAppApplnSolicitor().getEmail(),
-            notificationProperties.getJudgeForApprovedCaseApplicantEmailTemplate());
+        if (useDamageTemplate(caseData)) {
+            sendNotificationForJudicialDecision(caseData,
+                                                caseData.getGeneralAppApplnSolicitor().getEmail(),
```
Comment (smitatalawadekar-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java:346]: caseData.getGeneralAppApplnSolicitor().getEmail(),
can be fetched once before 'if' and the referred to in all places.

Comment (MMNycz) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java:346]: Good point - I update it as well in other part of code with the same situation.

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java (hunk 1 above)
Comment (MMNycz) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java:230]: I will refactor this part of Zikrur code - good point.

**Review by smitatalawadekar-hmcts — APPROVED** [review]

**Pull request merged**